virttest/asset.py

```diff
@@ -376,28 +376,38 @@ def get_file_asset(title, src_path, destination):
     return None
 
 
-def get_asset_info(asset):
+def get_asset_info(asset, ini_dir=None, section=None):
+    """"
+    Find .INI file and parse it. Retrun dictionary suitable for passing to
```

Comment:
How about `Parse $asset.ini file and returns a dictionary suitable for "asset.download_file`?